scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -107,13 +107,13 @@
     (write-status-to-datomic compute-cluster status)
     {:cook-expected-state :cook-expected-state/completed}))
 
-(defn handle-pod-preemption;
+(defn handle-pod-preemption
```

Comment:
Should we change the (running, waiting) case to use this new function as well?

Reply:
Ok.